ectweitmann/book-club-ui
# Feature/book details page cag

## What Features are Being Added?

- Added details to render in BookDetails.js
- Added componentDidMount to BookDetails.js 

## What is Being Refactored?

- Deleted extraneous files that we didn't need (React provided)

## Where should the reviewer start?

- BookDetails.js 

## How should this be manually tested?

- go to localhost:3000, click on the 'learn more' button to see single book details 

## What are the relevant tickets?

-  Build BookDetails component 

## Any Other Notes/Questions/Problems?

- None, y'all are great!!!
